decoders/connector/lansitec/nb-iot-container-tracker/v1.0.0/payload.js

```diff
@@ -0,0 +1,382 @@
+// nb-iot container tracker
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.